Tools/pyco/unpyc.py

```diff
@@ -212,7 +216,7 @@ def report(self):
         # We're on a roll! Print the code objects
         for i, offset in enumerate(self.code_offsets):
             reader.seek(offset)
-            values = struct.unpack("<14L", reader.read_raw_bytes(14 * 4))
+            values = struct.unpack("<16L", reader.read_raw_bytes(16 * 4))
```

Comment:
There are a few places in pyco.py itself where `<14L` needs to become `16L`, and also 56 needs to become 64 (these seem candidates for named constants. :-)
```diff
diff --git a/Tools/pyco/pyco.py b/Tools/pyco/pyco.py
index f26c0b022b..d99de9e92d 100644
--- a/Tools/pyco/pyco.py
+++ b/Tools/pyco/pyco.py@@ -693,9 +693,9 @@ def report(builder: Builder):
         except RuntimeError as err:
             print(f"  Code object {i} -- Error: {err}")
             continue
-        header = struct.unpack("<14L", b[:56])
+        header = struct.unpack("<16L", b[:64])
         n_instrs = header[-1]
-        bytecode = b[56 : 56 + 2 * n_instrs]
+        bytecode = b[64 : 64 + 2 * n_instrs]
         print(f"  Code object {i}")
         print(f"  Header {header}")
         stream = io.StringIO()
```
But there's another thing missing then, the Blob table looks like garbage.

Reply:
(Then again, in pyco/HEAD the blob table looks the same. I can't quite figure out where the blobs come from, but it's not in your PR. :-)

Reply:
(Oh, it's things like co_linetable. Ignore me for this part. But the diff above is still relevant. :-)

Reply:
This helped somewhat - calling the function still crashes but accessing attributes in its `__code__` no longer does.

Reply:
I mean it's not this change that helped, but the other changes in the same commit.  I made it not insert sub-constant to the builder.constants, and then we have the oparg carrying over like in the co_names case.